server/cluster.go

```diff
@@ -109,6 +109,8 @@ func (c *RaftCluster) start() error {
 	}
 	c.cachedCluster = cluster
 	c.coordinator = newCoordinator(c.cachedCluster, c.s.hbStreams, c.s.classifier)
+	regionStats := newRegionStatistics(c.s.scheduleOpt, c.s.classifier)
+	c.cachedCluster.bindRegionStatistics(regionStats)
```

Comment:
I think you can just `c.cachedCluster.regionStats = newRegionStatistics(c.s.scheduleOpt, c.s.classifier)`.